### PR TITLE
Update user map for June 2016 #5879

### LINKED ISSUE #5538: Update user map for May 2016
Follow #5163 
List of institutes to add are:

```
Australia   University of Melbourne
Australia   University of South Australia
Australia   Wesley College
Australia   TAFE Coomera
Australia   Swinburne University
Australia   Suzanne Cory High School
Belgium Vrije Universiteit Brussel 
Belgium Virgo Sapiens
Brazil  Associação Educacional Dom Bosco 
Brazil  Instituto Federal do Espírito Santo
Brazil  UniFOA
Canada  Simon Fraser University
Canada  Collège Regina Assumpta
Canada  University of New Brunswick
Canada  Georgian College
Canada  University of Manitoba
Ghana   Ashesi Unuiversity
Malaysia    Universiti Malaysia Sarawak
Malaysia    Universiti Sains Malaysia
Malaysia    University of Nottingham Malaysia Campus
Mexico  UNID
Netherlands Vrije Universiteit Amsterdam
Netherlands Teylingen College Leeuwenhorst
New Zealand University of Otago
Scotland    The Glasgow School of Art
Singapore   Singapore Institute of Technology
Singapore   Ngee Ann Polytechnic
UK  University of East London
USA Creighton University
USA Saint Mary's College
USA San Diego State University
USA New Mexico State University
USA Hopkins North Junior High School
USA Washington State University
USA Texas A&M University
USA Dawson high school
USA University of Texas at Arlington
USA Miami University
USA Louis Pasteur Middle School 67
USA University of Texas at Austin
USA North Carolina State University
USA Morgan State University
USA University of Louisville
USA Lehigh University
USA Carnegie Mellon University
USA University of Arizona South
USA Squalicum Hight School
USA BYU Idaho Online
USA Miller Career & Technical Center 
USA University at Buffalo
USA Newport News Schools 
USA Penn State University
USA University of Arizona
USA Calvin College
USA Tennessee Tech University
USA Allen High School
USA University of Kentucky
USA California College San Diego
USA University of Texas at Arlington
USA Kenan-Flagler Business School
```

### LINKED ISSUE #5879: Update user map for June 2016
Similar to #5158 
Here's the list.

```
Australia   Henley High School
Australia   Koonung Secondary
Australia   Charles Sturt University
Botswana    Alpha
Brazil  Unificados AEDB 2016
Brazil  UniFOA
Canada  Western University
Canada  Cooksville Creek P.S.
Canada  St. George's School
Canada  St. Andrew's College
Canada  Georgian College
cfdsfds ed
Egypt   Summits International School
France  IESEG School of Management
France  IESEG
Hungary Codecool Kft.
Indonesia   Swiss German University
Macedonia   University Ss. Cyril and Methodius
Malaysia    Universiti Teknologi Mara Johor
Malaysia    Taylors University
Mexico  ITESM
Mexico  Insituto Tecnologico de Estudios Superiores
Netherlands Eindhoven University of Technology
Netherlands The School of Life
Netherlands Universiteit Utrecht
New Zealand Auckland University
New Zealand Manukau Institute of Technology
New Zealand University of Auckland
Philippines New Era University
Serbia  University of Belgrade
Singapore   Nanyang Technological University
Singapore   Singapore Institute of Technology
Singapore   National University of Singapore
Singapore   Republic Polytechnic
Sweden  Stockholm University
Turkey  Meliksah University
UAE Hult International Business School
UK  Hult International Business School
UK  Hult International Business School
UK  Manchester Metropolitan University
UK  Hult International Business School
UK  Hult International Business School
UK  University of Central Lancashire
UK  University of Salford
UK  Southampton Solent University
UK  University of Bradford
USA University of Central Florida
USA Moreno Valley High School
USA Judson University
USA Texas A&M
USA East Grand Rapids Middle School
USA Georgia State University
USA Norwich Technical High School
USA Hult International Business School
USA University of Kansas Medical Center
USA Hult International Business School
USA Glenda Dawson High School
USA Rochester Institute of Technology
USA Lebanon High School
USA California State Polytechnic University, Pomona
USA Boston University 
USA Washington State University
USA Clark State Community College
USA Occidental College
USA Mary Dutkiewicz
USA Northeastern University
USA Bryant University
USA Indiana State University
USA Omada Health
USA Bryant University
USA Texas Woman's University 
USA Your Group Getaways
USA     Merrimack College
Venezuela   Centro de Desarrollo Araguaney
```

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #5538 and fixes #5879

~~Noticed that too many people wanting to do #5879 without even noticing this one, so working on this while we still got the time.~~ So that first timers can do #6107.

### CODE REVIEW

**Comment by junhaoyap** [comment]
On it

File: src/main/webapp/js/userMapData.json [hunk 1]
```diff
@@ -841,6 +874,7 @@
     "Southwestern Law School",
     "Spelman College",
     "Spring Hill College",
+    "Squalicum High School",
```
Comment (junhaoyap) [inline src/main/webapp/js/userMapData.json]: USA Squalicum Hight School, just checking to make sure "hight" is a typo

**Comment by junhaoyap** [comment]
I also assume ```"Swinburne University of Technology", === Australia   Swinburne University```

**Comment by junhaoyap** [comment]
Also ---

```
"Miami University",
"Miami University of Ohio",
```

same entities, or no?

**Comment by junhaoyap** [comment]
Also, we have ```Penn State University``` in the list and we have ```Pennsylvania State University``` in the json, which is the correct way of addressing the university?

**Comment by junhaoyap** [comment]
Amazing, you made sure to de-duplicate so let's sort through the comments we have and we can merge it soon

File: src/main/webapp/js/userMapData.json (hunk 1 above)
Comment (junhaoyap) [inline src/main/webapp/js/userMapData.json]: ```hight``` is from the list in the issue page btw

**Comment by wkurniawan07** [comment]
`Miami University (of Ohio)` is a strange case because there is a different `University of Miami`, but I'm not sure if the one without `of Ohio` refers to that. :thinking:

**Comment by damithc** [comment]
Sure, go ahead.

**Comment by wkurniawan07** [comment]
@junhaoyap more things to review :p

**Comment by wkurniawan07** [comment]
@junhaoyap can you review this (and the other PRs too) asap?

**Comment by junhaoyap** [comment]
Will do so tonight

**Comment by junhaoyap** [comment]
In the case of the Miami, I suppose they are 2 different universities and I assume both of them are using TEAMMATES :-)

**Comment by junhaoyap** [comment]
There is one ```Codecool Kft.``` as ```Codecool``` in the name list, which is correct?

**Comment by junhaoyap** [comment]
We are missing ```Netherlands Universiteit Utrecht``` it seems

**Comment by junhaoyap** [comment]
Good de-duplication on Auckland university, they are one and the same thing

**Comment by junhaoyap** [comment]
I believe ```Mary Dutkiewicz``` refers to a person, she can be found at https://www.linkedin.com/in/mary-dutkiewicz-7590ab1, althugh we're unsure where she is using it at

**Comment by junhaoyap** [comment]
I suppose ```Your Group Getaways``` stands for a group usage rather than a school entity usage, what is @damithc's stand on such usages?

**Comment by damithc** [comment]
`Your Group Getaways`: I'm not sure about it. Can be omitted.

**Comment by wkurniawan07** [comment]
@junhaoyap `Netherlands Universiteit Utrecht` and `Utrecht University` are one and the same. Any other concerns?

**Comment by junhaoyap** [comment]
Nope! Let's get it in!

**Comment by wkurniawan07** [comment]
@damithc this is ready for review, considering that #6122 is blocked this should be merged asap.

**Comment by damithc** [comment]
LGTM

**Pull request merged**